src/commands/deployment/__tests__/correlate.test.ts

```diff
@@ -56,15 +68,21 @@ describe('execute', () => {
       CI_PIPELINE_ID: '1',
       CI_JOB_ID: '1',
     }
-    const {context, code} = await runCLI(['--provider', 'argocd', '--dry-run'], envVars)
+    const {context, code} = await runCLI(
+      ['--provider', 'argocd', '--config-shas', 'abcdef', '--config-shas', 'fedcba', '--dry-run'],
```

Comment:
do we have a test where `--config-shas` is not passed but we _can_ retrieve the commits?

Reply:
Not at the moment. Is there an easy way to mock the git client? Otherwise the test would not work because we try to get the local commits that were not pushed.